src/constants.tsx

```diff
@@ -254,6 +254,7 @@ export const JURISDICTION_METADATA_RISK = 'jurisdiction_metadata-risk';
 export const JURISDICTION_METADATA_COVERAGE = 'jurisdiction_metadata-coverage';
 export const JURISDICTION_METADATA_POPULATION = 'jurisdiction_metadata-population';
 export const JURISDICTION_METADATA_TARGET = 'jurisdiction_metadata-target';
+export const JURISDICTION_METADATA_STUCTURES = 'jurisdiction_metadata-structures';
```

Comment:
Does the data warehouse query expect this to be `jurisdiction_metadata-structures` or `jurisdiction_metadata-structure`?  I think its helpful to provide a link to one of the queries that use this on the Reveal datawarehouse repo.

Reply:
It expects `jurisdiction_metadata-structures`

Reply:
My point is lets link that here to serve as documentation.

Reply:
Here is a link https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/32ac211b78a3d4cd3ada4f3f7e55024a06dd57b3/3-reveal/migrations/8-IRS-Lite/1-Zambia-2020/deploy/remove_hard_coded_jurisdiction_depth.psql#L154